src/app/components/header/header.component.css

```diff
@@ -0,0 +1,30 @@
+nav{
```

Comment:
* use formatting and proper indenting - as mentioned - install prettier and editorconfig for vscode - which will ensure you follow the correct indenting - two spaces or 4 spaces, and will allow you to format the entire file

Reply:
Yes, I have installed prettier after the first review.